### PR TITLE
Add test for add and remove functions

### PR DESCRIPTION
**Pull request by hiromataba**
In this project, we have tested the add and remove function

### CODE REVIEW

File: test/addRemove.test.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    todoAction.addTodo('Hiro');
+    const todos = JSON.parse(localStorage.getItem('List')) || [];
+    expect(todos.length).toBeGreaterThan(0);
+  });
+
+  it('should throw an error if no  is given', () => {
+    expect(() => {
+      todoAction.removeElement();
+    }).toThrow();
+  });
```
Comment (Dipeshtwis) [inline test/addRemove.test.js]: - [ ] Please add more test related to remove element like when we pass `taskid`. It will help your app to be good since, it will cover all the test and edge cases.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @vladanvi99 and @hiromataba, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: It has a test file (`[..].test.js`) for a file containing the **add item** and **delete item** functions that would be tested.
:+1: Uses mock for funtions that used `localStorage` , `DOM`.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [ ] README file should have step on how to test the app. Ex: `npm test` after cloning and installing.  Also mention how to install like `npm install`.

**_Also check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @hiromataba &  @vladanvi99. 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
